external/js/kyber/spec/pairing/point.spec.ts

```diff
@@ -26,6 +26,26 @@ describe("BN256 Point Tests", () => {
         expect(prop).toHold();
     });
 
+    it("should yield the same point by addition and multiplication", () => {
+        const prop = jsc.forall(jsc.uint8, (target) => {
+            const base = new BN256G1Point().base();
+            const scalarUnit = new BN256Scalar().one()
+            const pointUnit = new BN256G1Point().mul(scalarUnit, base)
+
+            const scalarAdder = new BN256Scalar();
+            const pointAdder = new BN256G1Point();
+            for (let i = 0; i < target; i++) {
+                scalarAdder.add(scalarAdder, scalarUnit)
+                pointAdder.add(pointAdder, pointUnit);
```

Comment:
`pointAdder.add(pointAdder.clone(), pointUnit);` fixes the test so it seems there is something wrong in the addition that modifies the first parameter.

Reply:
I'll have a better look tomorrow.